.github/workflows/windows-msys1.yml

```diff
@@ -234,6 +223,17 @@ jobs:
           name: testsuite-${{ matrix.target }}.log
           path: ${{ env.GITHUB_WORKSPACE }}/_build/tests/testsuite.log
 
+      - name: Install GnuCOBOL
```

Comment:
Any reason for swapping these steps ?

Reply:
> Any reason for swapping these steps ?

https://github.com/OCamlPro/gnucobol/pull/169#discussion_r1720904756

> Note that in debug mode, it has to be installed before running the testsuite, else you end up with tons of failures (don't ask me why...).

Ah…

Reply:
Just out of curiosity, is there an equivalent to `ulimit` on these systems?

Reply:
No idea...